app/models/build.rb

```diff
@@ -291,6 +291,10 @@ def is_running?
     IN_PROGRESS_STATES.include?(self.state)
   end
 
+  def as_json(options = {})
+    super(options).deep_merge!('build' => {'elapsed_time' => elapsed_time})
```

Comment:
could this have been accomplished with the `methods` key passed in options to as_json? Docs: http://api.rubyonrails.org/classes/ActiveModel/Serializers/JSON.html